src/ondil/distributions/mv_t_low_rank.py

```diff
@@ -468,3 +468,284 @@ def mv_t_lr_partial_2_dof(y, mat_mu, mat_d, mat_v, dof):
     )
     part2 = (Z * (dof * Z - k * (2 * dof + Z))) / (2 * dof**2 * (dof + Z) ** 2)
     return part1 + part2
+
+
+class MultivariateStudentTInverseLowRank(MultivariateDistributionMixin, Distribution):
```

Comment:
Why do we have this redefenition here?
Please:
- rename this class or
- rename the first class,
- remove one of them.